### ISSUE TITLE
NVDA Screen Reader Unable to Read Custom Error Messages

### ISSUE THREAD

**Issue by rpw1** [ngx-formly/ngx-formly#3737]
[Here is the demo I created on stackblitz using bootstrap template](https://ngx-formly-ui-bootstrap-qjfo3p.stackblitz.io)

Using tabs and a screen reader, it will never read out the issue when accessing the input box. It doesn't read out that the input is invalid when showing an error or the error message itself.

It seems that the form is missing the aria-invalid, arai-required, and aria-describedby.
![image](https://github.com/ngx-formly/ngx-formly/assets/47835461/722f505d-fc52-44dc-a529-bd9247c10978)

Is there a way to add/update these attributes with the current version of formly?

I have tried to do a work around by adding aria-invalid and aria-description to be updated expressions
```
this.formlyField.expressions = {
    'props.attributes.aria-description': 'formState.currentValidationErrorMessage',
    'props.attributes.aria-invalid': 'formState.isInvalid'
}
```
Then I tried adding formState in the FormlyFieldConfig.FormlyFormsOptions where I can update the values.
```
this.formlyField.options.formState = {
    'currentValidationErrorMessage': '',
    'isInvalid': false
}
```
Finally I tried updating the states in the validator, but the items remain stagnant with aria-invalid set to true and aria-description set to the minDAteValidationMessage
```
minDate: {
    expression: (c: UntypedFormControl, field: FormlyFieldConfig) => {
        let isValid: boolean = !field.props?.attributes?.min || this.formatDate(c.value) >= field?.props?.attributes?.min
        if (field.options) {
          field.options.formState.currentValidationErrorMessage = isValid ? '' : FormlyErrorMessages.minDateValidationMessage(null, field);
          field.options.formState.isInvalid = isValid ? 'false' : 'true';
         }
        return isValid
    },
    message: FormlyErrorMessages.minDateValidationMessage
},
```

I would appreciate any help I can get. I know adding aria attributes was done in #2458, so I hope it isn't too bad to do for bootstrap as well.

Formly version: 6.1.1
Screen reader: NVDA

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.2.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.2.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
